content/applications/websites/ecommerce/checkout_payment_shipping/checkout.rst

```diff
@@ -0,0 +1,144 @@
+========
+Checkout
+========
+
+You can customize the checkout process by enabling **additional features** for the different
+checkout steps, adding additional content through the **website builder** tool, and enabling other
+features such as **express checkout**, or **sign in/up at checkout**.
+
+Checkout steps
+==============
+
+Review Order
+------------
+
+Promo code
+~~~~~~~~~~
+
+If you have enabled :guilabel:`Discounts, Loyalty, & Gift Card` in the settings
+(:menuselection:`Configuration --> Settings --> Shop - Products`), you can enable a
+:guilabel:`Promo Code` field (:menuselection:`Edit --> Customize`) from any checkout page.
+Customers can then redeem gift cards and promotional codes at the :guilabel:`Review Order` step.
+
+.. image:: checkout/checkout-promo.png
+   :align: center
+   :alt: Gift card and promotion field
+
+Discount subtotal
+~~~~~~~~~~~~~~~~~
+
+If :guilabel:`Discounts, Loyalty, & Gift Card` is enabled, you can display the **subtotal** after
+discount at the :guilabel:`Review Order` step. From any checkout page, go to :menuselection:`Edit
+--> Customize` and enable :guilabel:`Show Discount in Subtotal`.
+
+.. image:: checkout/checkout-subtotal.png
+   :align: center
+   :alt: Subtotal discount
+
+Address
+-------
+
+B2B fields
+~~~~~~~~~~
+
+Optional :guilabel:`TIN/VAT` and :guilabel:`Company Name` fields can be added to the
+:guilabel:`Billing Address` form for B2B customers. To add the fields, go to :menuselection:`Edit
+--> Customize` from any checkout page, and enable :guilabel:`Show B2B fields`.
+
+Request extra info (extra step)
+-------------------------------
+
+You can request :guilabel:`Extra Info` from the customer by adding an **extra step** between the
+:guilabel:`Address` and :guilabel:`Confirm Order` steps. The :guilabel:`Extra Info` step is an
+online form linked to the quotation or sales order of the customer. The information added in the
+extra step is available on the quotation or sales order of the customer from the back end, in the
+**Sales** app.
```

Comment:
```suggestion
You can request :guilabel:`Extra Info` from the customer by adding an **extra step** between the
:guilabel:`Address` and :guilabel:`Confirm Order` steps. To do so, go to :menuselection:`Edit
--> Customize` from any checkout page, and enable : :guilabel:`Extra Step Option`
The :guilabel:`Extra Info` step is an online form linked to the quotation or sales order of the customer. The information added in the extra step is available on the quotation or sales order of the customer from the back end, in the **Sales** app.
```

Reply:
I did add a picture to make it more understandable where that extra page can be found in the process, but not the steps. I don't think it is necessary.